### ISSUE TITLE
Keyboard and emoji both open simultaneously  in android P

### ISSUE THREAD

**Issue by samclam** [hani-momanii/SuperNova-Emoji#45]
### Best emoji library I ever used for android. Thanks for making this.

Although I'm facing few issues with latest version. While clicking on emoji icon both android keyboard and emoji keyboard opens simultaneously. 

Android Version : P
Here is the screenshot from my end

![screenshot_1528897736](https://user-images.githubusercontent.com/30346069/41357779-89252a0c-6f44-11e8-999e-685177a0473b.png)

**Comment by Vanthiyadevan** [comment]
Yes I'm facing this problem too on lastest developer android preview.

**Comment by JeyRia** [comment]
i have the same problem in the latest android version P , how to i resolve this problem.

**Comment by amitmad** [comment]
using this

emojIcon.setKeyboardListener(new EmojIconActions.KeyboardListener() {
@Override
public void onKeyboardOpen() {
    Log.e("Keyboard","open");
  }

@Override
public void onKeyboardClose() {
  Log.e("Keyboard","close");
}
});

**Comment by andysagar** [comment]
got any solution for that?
